packages/clerk-js/src/ui/elements/Section.tsx

```diff
@@ -49,14 +49,13 @@ const ProfileSectionRoot = (props: ProfileSectionProps) => {
         elementId={descriptors.profileSectionContent.setId(id)}
         gap={2}
         ref={ref}
-        sx={t => ({
-          paddingLeft: t.space.$6,
```

Comment:
This will probably cause the component to be off with designs as we've already aligned it. Why are we removing this?

Reply:
I have made some changes to align to the designs without the need of this.